### PR TITLE
Add topic creation properties for MongoDB source connector

### PR DESCRIPTION
**Pull request by tomasz-sadura**

### CODE REVIEW

**Review by bochenekmartin — APPROVED** [review]

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc [hunk 1]
```diff
@@ -129,6 +129,18 @@ on the *Create Connector* page:
 | `Offset partition name`
 | `offset.partition.name`
 | Use to specify a custom offset partition name. If blank, the default partition name based on the connection details is used.
+
+| `Topic creation enabled`
+| `topic.creation.enable`
+| Specifies whether or not to allow automatic creation of topics. Default is enabled.
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc]: Is the default value actually `enabled` or `true`? Or just by having the line, the configuration is changed to enable?

Comment (tomasz-sadura) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc]: the actual is `topic.creation.enable=true`

Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc]:
```suggestion
| Specifies whether or not to allow automatic creation of topics. Default is `true`.
```

**Review by Deflaimun — APPROVED** [review]

**Comment by Deflaimun** [comment]
Please check https://github.com/redpanda-data/docs/pull/276

**Pull request merged**